docs/fast_data/faq/bootstrapping-fast-data.md

```diff
@@ -0,0 +1,154 @@
+:::
+
+### What should be the required resources to perform a Single View Refresh?
+
+The **Single View Refresh** is an operation performed to update all the projection changes of a single view: this activity is needed during the first release of a single view and every time a new feature is released in the environment, since it will start the aggregation pipeline of the [**Single-View Creator**. (**SVC**)](/fast_data/single_view_creator.md)
+
+On a [Standard Architecture](/fast_data/architecture.md#standard-architecture), this means that the database will perform operations on:
+
+* the projection changes collections, because the **SVC** will update the projection changes record updating/removing the `changes` record; 
+* the projections that are used by the single view aggregation mechanism; 
+* the single view collections, to insert/update/delete records corresponding to single views.
+
+:::tip Execution Time
+The time generation is **strictly dependent** on the number of projections that are needed to compute all the fields of a single view. 
+
+On a single view having three projections and custom functions, a refresh operation involving 16k projection changes records required _5 minutes_, which is approximately a rate of `1k single view/minute`, using up until the 80% of a single MongoDb core.
+:::
+
+On an [Event-Driven Architecture](/fast_data/architecture.md#event-driven-architecture), projection changes collection is removed in favour of trigger topics, so Fast Data will use less database resources.
+
+
+### Why is the strategy execution or the single view generation slow?
+
+If the execution of a strategy or the aggregation of a single view takes too much time, it means that maybe some index is missing in the collection involved. You should check that:
+
+* projections have all the queries covered by indexes;
+* the projection changes collection, if used, has [the required index](/fast_data/configuration/realtime_updater.md#custom-projection-changes-collection);
+* the single view collection has a unique index that is equal to the [Single View Key configuration](/fast_data/configuration/config_maps/singleViewKey.md).
+
+## Fast Data Plugins
+
+### How should I set the resource limits of the Real-Time Updater (RTU)?
+
+On a [Standard Architecture](/fast_data/architecture.md#standard-architecture), the RTU performs strategies to generate projection changes: based on the complexity of this strategy and the number of projections involved, this could require a variable range of resources to use.
+
+Also, keep in mind that you may need to have a number of replicas greater than 1 as the number of projections and the corresponding number of partitions of their topics increases over time.
+
+:::tip Use case
+A RTU having two projections with strategies enabled, can reach up until an ingestion rate of `10k message/minute` with one replica having the following setting:
+
+* CPU:
+  * `200m` request
+  * `400m` limit
```

Comment:
Just to update on this number, after some internal tests we set the default limit for the RTU plugin to `600m`

Reply:
hi @danibix95 i've updated this reference